Jerkinator/SecondRollFrontend
# 45 connect searchbar to backend

To test this out:
1. git fetch in main.
2. git checkout 45-connect-searchbar-to-backend
3. Go to the home page and you should see the search bar. 
4. Enter the title of a game that you know exists in the database. You don't have to use a capital letter, but make sure you spell it as it is spelled in the database. 
5. If successful, the result should appear with the corresponding title next to the searchbar as clickable blue text. If there are multiple games with the same name (Monopol), both will display. 
6. When clicked, it should redirect you to the specific game info. 

Not as fancy as the search-function I attempted before, but this one loads much quicker, and does not display an annoying list of results as soon as the page is loaded/reloaded.
